index.ts

```diff
@@ -134,86 +122,40 @@ export class U64Array extends Uint64Array {
   toString(): string {
     return typeConversion.u64ArrayToString(this)
   }
-
-  toAddress(): Address {
-    return typeConversion.u256ToH160(this) as Address
-  }
-}
-
-/** An Ethereum address (20 bytes). */
-export class Address extends ByteArray {
-  static fromString(s: String): Address {
-    return changetype<Address>(typeConversion.stringToH160(s))
-  }
 }
 
-/** An arbitrary size integer. */
-export class BigInt extends ByteArray {}
-
 /** A dynamically-sized byte array. */
-export class Bytes extends ByteArray {}
-
-/** A 160-bit hash. */
-export class H160 extends ByteArray {
-  static fromString(s: String): H160 {
-    return typeConversion.stringToH160(s)
-  }
+export class Bytes extends ByteArray {
+  constructor() {}
```

Comment:
What's the purpose of the empty constructors?

Reply:
Without these, we can't do `new Bytes()` anywhere. We may not want to though — I can double-check if this is really needed with the new code generation in Graph CLI.

Reply:
Ah got it. It was more of a language curiosity, it's probably a useful thing to have.